actions/views/create_comment.jsx

```diff
@@ -77,7 +78,14 @@ export function submitPost(channelId, rootId, draft) {
             create_at: time,
         };
 
-        dispatch(PostActions.createPost(post, draft.fileInfos));
+        const hookResult = await dispatch(runMessageWillBePostedHooks(post));
```

Comment:
We should handle the case `!hookResult` to avoid crashing the page here.
To be consistent with the behavior of the server side hook. I would say `!hookResult` should not modify the post and also not reject it.

Reply:
I'll add in the null check, but I don't think we can ensure that `!hookResult` means that the post was not modified without making a copy of the post before calling each hook

Reply:
Fixed in `runMessageWillBePostedHooks` so it will always return a defined object now.